### PR TITLE
Complete list of systems on the first page

### PR DESCRIPTION
**Pull request by addshore**
This comes from a Google doc (with slighly name changes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +9,11 @@ This documentation is made up of various [arc42 structured](https://docs.arc42.o
...
 - [WikibaseClient](./systems/WikibaseClient/)
-- TBA...
+- WikibaseLexeme
+- EntitySchema
+- QualityConstraints
+- Article Placeholder
+- Property Suggester
+- Termbox SSR
+- Query UI
+- Query Builder
```
Comment (jakobw) [inline README.md:19]: Hm. We've had this discussion a few times now, and I still don't see these on the same level. Except maybe for the query service, none of the new additions even fit the *system* label IMO.

Comment (addshore) [reply README.md:19]: Yeah, did we write those notes down anywhere? I don't see them in the Google doc...

Comment (jakobw) [reply README.md:19]: No, I don't think we did. Ughhhh.

Comment (tzhelyazkova) [reply README.md:19]: Yes, from what I remember, we agreed that the top components are repo, client and query service.
Everything else is at least one level down from there.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,22 @@ This documentation is made up of various [arc42 structured](https://docs.arc42.o
...
+Multiple top level systems are identified.
+
+Some of these also have fairly notable sub systems identified here for convenience.
+
 - [WikibaseRepo](./systems/WikibaseRepo/)
+  - Termbox & Termbox SSR
+  - EntitySchema
+  - Property Suggester
+  - QualityConstraints
+  - WikibaseLexeme
```
Comment (Silvan-WMDE) [inline README.md:20]: as per our discussion notes, `MediaInfo` should be added here under WikibaseRepo

**Review by Silvan-WMDE — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (addshore) [reply README.md:20]: Added

**Review by jakobw — APPROVED** [review]

**Review by tzhelyazkova — APPROVED** [review]
nice

**Pull request merged**